devices.js

```diff
@@ -2627,6 +2627,28 @@ const devices = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['Remote Control N2'],
+        model: 'W2049',
+        vendor: 'IKEA',
+        description: 'Remote Control N2',
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.E1524_E1810_arrow_click,
```

Comment:
@Koenkk perhaps the E1524_E1810_ prefix need to be renamed to E1524_E1810_W2049_ or maybe something more generic like ikea_ ?

Reply:
Good point, done